modules/dmrpp_module/SuperChunk.h

```diff
@@ -55,44 +52,47 @@ class SuperChunk {
     bool d_is_read;
     char *d_read_buffer;
 
+    bool d_uses_fill_value{false};
+
     bool is_contiguous(std::shared_ptr<Chunk> candidate_chunk);
     void map_chunks_to_buffer();
     void read_aggregate_bytes();
+    void read_fill_value_chunk();
 
 public:
-
+    // TODO Make the sc_id an uint64 and not a string - the code uses sstream to make the value. jhrg 5/7/22
```

Comment:
but... What if we end up using a readable name? Will the be useful down the road?

Reply:
I don't understand - my comment is that we don't need a string here and doing so uses more time than not. An int will be fine and if we need a string lagter on, we do that. 

I'm not changing the code now; this can go in a general TODO ticket and we can revisit the issue.